repair/repair.cc

```diff
@@ -743,11 +743,11 @@ static future<tasks::task_manager::task_ptr> start_repair_task(tasks::task_manag
 }
 
 future<> range_repair_task_impl::run() {
-    // TODO: implement
-    return make_ready_future<>();
+    return _ri->repair_range(_range, _table_id);
```

Comment:
To be honest, I am not a big fan of adding tons of foo_task_impl. They are just too many of them which makes it is less useful to for user to track them. 

We  have shard_repair_task_impl and range_repair_task_impl. 
Say we have a repair task with 20 tables and 1024 ranges and 32 shards. 

We are going to create

20 * 1024  * 32 shard_repair_task_impl

and 

20 * 1024  range_repair_task_impl.

There is no way for a user to track this amount of tasks.

Reply:
Maybe we can add an attribute to the task class to show/hide it in the tasks list.
I think it could be helpful to use the task_manager::task class for internal purposes even if we need not show it over the api.

Reply:
something like `using internal = bool_class<struct internal_tag>`